source/lex.tex

```diff
@@ -1845,6 +1817,79 @@
 The effect of attempting to modify a \grammarterm{string-literal} is undefined.
 \end{note}
 
+\pnum
+String literal objects are initialized with
+the sequence of code unit values
+corresponding to the \grammarterm{string-literal}'s sequence of
+\grammarterm{s-char}s (for a non-raw string literal) and
+\grammarterm{r-char}s (for a raw string literal)
+in order as follows:
+\begin{itemize}
+\item
+The sequence of characters denoted by each contiguous sequence of
+\grammarterm{basic-s-char}s,
+\grammarterm{r-char}s,
+\grammarterm{simple-escape-sequence}s\iref{lex.ccon}, and
+\grammarterm{universal-character-name}s\iref{lex.charset}
+is encoded to a code unit sequence
+using the \grammarterm{string-literal}'s associated character encoding.
+If a character lacks representation in the associated character encoding, then:
+\begin{itemize}
+\item
+If the \grammarterm{string-literal}'s \grammarterm{encoding-prefix}
+is absent or \tcode{L},
+then the \grammarterm{string-literal} is conditionally-supported and
+an
+\impldef{code unit sequence for non-representable \grammarterm{string-literal}}
+code unit sequence is encoded.
+\item
+Otherwise, the \grammarterm{string-literal} is ill-formed.
+\end{itemize}
+When encoding a stateful character encoding,
+implementations should encode the first such sequence
+beginning with the initial encoding state and
+encode subsequent sequences
+beginning with the final encoding state of the prior sequence.
+\begin{note}
+The encoded code unit sequence can differ from
+the sequence of code units that would be obtained by
+encoding each character independently.
+\end{note}
+\item
+Each \grammarterm{numeric-escape-sequence}\iref{lex.ccon}
+that specifies an integer value $v$
+contributes a single code unit with a value as follows:
+\begin{itemize}
+\item
+If $v$ does not exceed the range of representable values of
+the \grammarterm{string-literal}'s array element type,
+then the value is $v$.
+\item
+Otherwise,
+if the \grammarterm{string-literal}'s \grammarterm{encoding-prefix}
+is absent or \tcode{L}, and
+$v$ does not exceed the range of representable values of
+the corresponding unsigned type for the underlying type of
+the \grammarterm{string-literal}'s array element type,
+then the value is the unique value of
+the \grammarterm{string-literal}'s array element type \tcode{T}
+that is congruent to $v$ modulo $2^N$, where $N$ is the width of \tcode{T}.
+\item
+Otherwise, the \grammarterm{string-literal} is ill-formed.
+\end{itemize}
+When encoding a stateful character encoding,
+these sequences should have no effect on encoding state.
+\item
+Each \grammarterm{conditional-escape-sequence}\iref{lex.ccon}
+contributes an
+\impdef{code unit sequence of \grammarterm{conditional-escape-sequence}}
```

Comment:
Using the grammerterm inside the impldef like this rendered as "/implementation-defined/code unit sequence of ..." (with implementation-defined in italics and the impldef text afterwords).

Reply:
fixed